app/src/main/java/org/gdg/frisbee/android/common/GdgActivity.java

```diff
@@ -114,14 +114,16 @@ public void onCreate(Bundle savedInstanceState) {
     }
 
     protected void createGoogleApiClient() {
-        mGoogleApiClient = new GoogleApiClient.Builder(this)
-                .addApi(Plus.API)
-                .addApi(Games.API)
-                .addApi(AppStateManager.API)
-                .addScope(Plus.SCOPE_PLUS_LOGIN)
-                .addScope(Plus.SCOPE_PLUS_PROFILE)
-                .addScope(Games.SCOPE_GAMES)
-                .build();
+        GoogleApiClient.Builder builder = new GoogleApiClient.Builder(this)
+                .addApiIfAvailable(AppStateManager.API);
+        if (PrefUtils.isSignedIn(this)) {
+            builder.addApiIfAvailable(Plus.API, Plus.SCOPE_PLUS_LOGIN, Plus.SCOPE_PLUS_PROFILE)
```

Comment:
I don't think we had a problem with signed out users. Signed out users can see all the data and the pictures from Google+. It's because we are using Google+ java api to get them.